reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -1309,25 +1309,25 @@ public RequestSender request(HttpMethod method) {
 	}
 
 	/**
-	 * Specifies the response timeout duration in milliseconds.
-	 * This is time that takes to receive a response after sending a request.
+	 * Specifies the response timeout duration.
```

Comment:
What do you think about this?

```suggestion
	 * Specifies the maximum interval allowed between read operations (resolution: ms). {@link io.netty.handler.timeout.ReadTimeoutHandler} is added to the channel pipeline after sending the request and is removed when the response is fully received. 
```

Reply:
LGTM.